vector-im/element-android
# Fixes /upgraderoom command not doing anything

## Type of change


## Content

Fixes the `/upgraderoom` command not doing anything due to a posted live data event getting overridden before it's observed

## Motivation and context



Closes https://github.com/vector-im/element-android/issues/6153

## Screenshots / GIFs



| Before | After |
|-|-|
| https://user-images.githubusercontent.com/20701752/170278411-d41ca380-5176-484d-92f2-f682fc0d6d3b.mp4 | https://user-images.githubusercontent.com/20701752/170278455-82228685-0104-4572-aa77-63ed7dd93214.mp4 |

## Tests



- Enable developer mode in advanced settings
- Go to a room and enter `/upgraderoom [anyNumber]`
- See upgrade bottom sheet now pops up

## Tested devices

- OS version(s): Android 12

## Checklist
